pages/maintainers.vue

```diff
@@ -1,5 +1,6 @@
 <template>
-  <section>
+  <section class="maintainers">
+    <CommonSwitchButton />
     <CommonGrid :maintainers="maintainersList" data-cy="maintainers" />
   </section>
```

Comment:
💬 
What about wrapping this inside a `<transition>`, maybe that would make the switch to look a bit smoother? WDYT>

Reply:
We can add transitions to all pages to make overall change smoother, but I was thinking on doing this later over the motion process. I'll add it as a task so it can be added once we have all pages.